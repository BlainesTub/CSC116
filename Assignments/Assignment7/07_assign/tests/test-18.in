(exp(x) + log(x) + sin(x) + cos(x))/2.0
3.0