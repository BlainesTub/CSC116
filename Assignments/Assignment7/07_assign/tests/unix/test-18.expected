f(x) = (exp(x) + log(x) + sin(x) + cos(x))/2
f(3) = 10.1676
f'(x) = ((exp(x)*1 + 1/x + cos(x)*1 + -1*sin(x)*1)*2 - (exp(x) + log(x) + sin(x) + cos(x))*0)/2^2
f'(3) = 9.64388
